lib/ansible/modules/packaging/os/urpmi.py

```diff
@@ -192,13 +190,19 @@ def main():
             update_cache=dict(type='bool', default=False, aliases=['update-cache']),
             force=dict(type='bool', default=True),
             no_recommends=dict(type='bool', default=True, aliases=['no-recommends']),
-            package=dict(type='str', required=True, aliases=['name', 'pkg']),
+            package=dict(type='list', required=True, aliases=['name', 'pkg']),
             root=dict(type='str', aliases=['installroot']),
         ),
     )
 
-    if not os.path.exists(URPMI_PATH):
-        module.fail_json(msg="cannot find urpmi, looking for %s" % (URPMI_PATH))
+    global URPMI_PATH
```

Comment:
Rather than making these globals, store them as locals and pass them into the functions, or just run `mode.get_bin_bath` within the function. Globals are not worth the future troubleshooting complexity in this case, especially since `RPM_PATH` is the only variable used in two different functions.